Formula/watchman.rb

```diff
@@ -23,7 +29,8 @@ def install
                           "--with-pcre",
                           # we'll do the homebrew specific python
                           # installation below
-                          "--without-python"
+                          "--without-python",
+                          "--enable-statedir=#{var}/run/watchman"
```

Comment:
Smells like it's worth hunting why we change permissions on bottled `var` and whether doing so has any downsides.